### DISCUSSION TITLE
AdminLTE v3 installs bootstrap 3...

### DISCUSSION THREAD

**Discussion by dima2306** [ColorlibHQ/AdminLTE#4095] [Q&A]
Hello everyone,

I'm installing AdminLTE via `npm install admin-lte@^3.1.0 --save` but it installs bootstrap v3.4.1 I think because of bootstrap-switch peer dependency. How can I force npm to install bootstrap 4 not including it explicitly in package.json?

And why on earth does it try to install bootstrap v3 when the template is using v4?

Or maybe issues are on my side and I'm doing something wrong?

**Comment by dima2306** [comment]
No answer, really?

**Reply by REJack** [reply]
Next time first search for similar issues before posting a new one, or even a questions.
#2640

**Reply by dima2306** [reply]
I've searched for the issues before opening a new discussion but didn't find a solution so posed here in the hope of the answer and I got it. Thanks.

But the problem is that I'm importing bootstrap.scss in my scss file, so I'm not using the dist folder to compile it and thus, every time I update other packages, npm if I have boostrap4 it reverts to 3.

Currently, I'm forcing npm to use bootstrap 4...

**Reply by REJack** [reply]
that's curious we doesn't ran into this problem, but with AdminLTE v4 we will drop all jQuery Plugins so this issue will gone too.

**Reply by dima2306** [reply]
Yeah, that's strange but it's fact.
Great news about AdminLTE v4 dropping jQuery support. Looking forward to trying it out.
